torch/csrc/jit/codegen/cuda/compute_at.cpp

```diff
@@ -119,12 +124,42 @@ unsigned int getReplayablePosPasC(
               return mappable_roots.find(root_id) == mappable_roots.end() &&
                   c2p_root_map.find(root_id) != c2p_root_map.end();
             })) {
+      max_consumer_pos = consumer_pos - 1;
       continue;
     }
-    return consumer_pos;
+    break;
```

Comment:
Quick question: if `break` is necessary here, is the loop itself really necessary?

Reply:
Sorry, just my confusion. Never mind.